README.md

```diff
@@ -405,8 +406,18 @@ Inspired by [30 seconds of code](https://github.com/Chalarangelo/30-seconds-of-c
       }
     }
   }
+  
+```
+### Length of File
+
+```java
+    public static long getLengthofFile(String filename) throws IOException {
+        File f = new File(filename);
+        return f.length();
```

Comment:
I think this is legacy Java. Nowadays a better way would be to use `Files.size()` method.